src/sentry/search/events/fields.py

```diff
@@ -406,6 +407,7 @@ def normalize_count_if_value(args: Mapping[str, str]) -> Union[float, str]:
             "timestamp.to_day", "timestamp.to_day", expression=["toStartOfDay", ["timestamp"]]
         ),
         PseudoField(ERROR_UNHANDLED_ALIAS, ERROR_UNHANDLED_ALIAS, expression=["notHandled", []]),
+        PseudoField(ERROR_HANDLED_ALIAS, ERROR_HANDLED_ALIAS, expression=["isHandled", []]),
```

Comment:
Also error.handled has been returning values in arrays and adding this pseudo field just returns the value, hence some of the changes to `tests/snuba/api/endpoints/test_organization_events_v2.py` etc. I checked field renderers for error.handled, they handle both array and just the value, so it shouldn't break any rendering. We also don't return the error.handled field anywhere in the code afaict, so this *shouldnt* break anything...

Reply:
I feel like I'm missing something 🔍

Reply:
From slack, 
here's additional context:
- https://github.com/getsentry/sentry/pull/21256
- https://www.notion.so/sentry/Crashes-in-SDKs-and-Product-fa249e31d6274b66b1bfff966e00b668

and while rendering doesn't change, let's maintain this quirk of our current search syntax, and revisit this later if need be, rather than sliding this into the snql changes

Reply:
We currently don't have a way to alias columns (there's a slack thread about it, looks like there's a task for it), error.handled could be easily implemented with aliasing columns, so I put in a [hack](https://github.com/getsentry/sentry/pull/27330/commits/b336212b0448b38653499cd65511eada3cae983f#diff-e128f3ece0d99c8b156b972bf9571c6b48ce8ccf6d478284f739838e68d813b9R139-R142) and took it out. From Tony:
>I'm looking at the snql docs for Column and was wondering if there's a reason why it doesnt support alias like Function. Right now we have columns like group_id that we have translate back into issue.id for the user in sentry, and it would be great if we can specify an alias like we already do for aggregates and skip this extra post processing step in sentry.

Reply:
Added back with hack